core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -68,7 +68,7 @@ private Cache() {}
        * @param ufsConf the ufs configuration
        * @return the UFS instance
        */
-      UnderFileSystem get(String path, Object ufsConf) {
+      UnderFileSystem get(String path, Map<String, String> ufsConf) {
```

Comment:
do you mean create a class sth like `UfsConf`, which is simply a wrapper of `Map<String, String>`?